news/2019-02-22-acknowledging-community-contributors.md

```diff
@@ -0,0 +1,211 @@
+
+## Development
+
+### ekr
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/ekr.jpg)](https://osu.ppy.sh/users/4497706)
+
+Responsible for considerable contributions to the osu!lazer project, ekr's efforts have focused foremost on reviewing code and making grand steps towards development of the game's input system, essentially rewriting it from scratch.
+
+Noted specifically as one of the few developers brave enough to delve into the core architecture of osu!'s framework, ekr's work ethic and willingness to get involved is highly valued.
+
+## Community
+
+### HappyStick
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/HappyStick.jpg)](https://osu.ppy.sh/users/256802)
+
+All but leaving behind a successful streaming career and tournament series of his own, HappyStick's jaunt into the deeper affairs of the community has proven instrumental in the continuation of the osu! World Cup series in the wake of its previous organizer's farewell.
+
+Beyond his award-winning smile, HappyStick has proven himself to be an invaluable addition to the osu!team, heading the new osu! Coffee Hour series, generally spending many sleepless nights setting up and streaming World Cup matches, creating tools to ensure it is easier for others to run them, and so much more.
+
+We could say many things about HappyStick, but foremost among them is how thankful we are for his continued contribution.
+
+### juankristal
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/juankristal.jpg)](https://osu.ppy.sh/users/443656)
+
+Something of a living meme among the osu! tournament scene, juankristal has been perpetually blamed for everything and anything that has gone both wrong and right for countless years.
+
+For once, we are delighted to firmly pin the blame on him for the continuation of the osu! World Cup series as a whole, stepping up to lead the INTENSE organization required to schedule, oversee and generally maintain multiple multinational tournaments.
+
+The workload involved in running the osu! World Cups is *immense*, perhaps far greater than most people realize, and juankristal has not only stepped up himself to do it out of a sheer desire to see it done, but has *excelled* at it, allowing the series to continue without so much as a blip.
+
+### WalterToro
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/WalterToro.jpg)](https://osu.ppy.sh/users/5281416)
+
+A mainstay force within the Global Moderation Team, WalterToro is not your ordinary chat moderator. He is *everywhere*, in the osu!wiki scene, in the tournaments scene, and wildly active in the Spanish community to say the least.
+
+From refereeing matches to refereeing people, we are intensely thankful for WalterToro's incredible work ethic and tireless drive to do what needs to be done.
```

Comment:
intensely thankful seems like a rather odd way to be thankful imo

Reply:
altered